### ISSUE TITLE
Tensorboard Error 404, path /[[_dataImageSrc]] not found

### ISSUE THREAD

**Issue by GustavZ** [tensorflow/tensorflow#16980]
I am using the object detection API
and training a new ssd_mobilenet from scratch with my dataset.
I already did a successful retraining, but now i want to try a complete new training without checkpoint.

The training and evaluation job seem to run normal, but when i start tensorboard and open port 6006 in my browser,  the terminal where its running shows following error:
`
W0213 11:04:23.869396 Thread-2 application.py:273] path /[[_dataImageSrc]] not found, sending 404
`
and no scalars, except of the learning rate, are visualized in tensorboard.
In contrast to that Evaluation images, the graph, distributions and histogram are all shown correctly.

**Comment by GustavZ** [comment]
although the error is still shown, now everything works and is displayed as it should.
Idk what was the reason but doesn't seem to be really problematic.
Thanks, closing!
